tests/e2e/utils/deleteMedia.js

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * WordPress dependencies
+ */
+import { visitAdminPage } from '@wordpress/e2e-test-utils';
+
+/**
+ * Creates a new story.
+ *
+ * @param {string}fileName File name to delete with ext.
+ */
+async function deleteMedia(fileName) {
+  await visitAdminPage('upload.php', 'mode=list');
+
+  await expect(page).toMatch(fileName);
+
+  await expect(page).toClick('a', { text: fileName, exact_text: true });
+  await page.waitForNavigation();
```

Comment:
I'd usually do

```js
await Promise.all([
  expect(page).toClick('a', { text: fileName, exact_text: true }),
  page.waitForNavigation()
]);
```

to prevent the case where we `waitForNavigation` if it has already happened since the click.

Reply:
☺️

Reply:
Fixed in https://github.com/google/web-stories-wp/pull/5849/commits/a5f7667e9c4e06126df3349c6ef2ca290997040d